cms/static/js/views/settings/main.js

```diff
@@ -366,6 +366,8 @@ define(['js/views/validation', 'codemirror', 'underscore', 'jquery', 'jquery.ui'
                                cachethis.setAndValidate(field, newVal);
                            }
                        });
+                       cmTextArea = this.codeMirrors[thisTarget.id].getInputField();
+                       cmTextArea.setAttribute('id', 'cm-textarea');
```

Comment:
I did not think where was an xblock ID here.  I will look again for a unique ID in the python file for the settings area.